engine/glfw/lib/ios/app/ViewController.m

```diff
@@ -18,7 +18,7 @@
 #import "EAGLView.h"
 
 extern int g_IsReboot;
-static int g_view_type = GLFW_OPENGL_API;
+static int g_view_type = GLFW_NO_API;
```

Comment:
Defaulting to Vulkan (Metal) lets us avoid adding the dependency of the glfw library for the graphics_vulkan.cpp. This is good since we don't want to use that for Switch.